warehouse/models/staging/payments/_stg_payments.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+
+models:
+  - name: stg_payments__elavon_transactions
+    description: Transactions processed by Elavon
+    columns:
```

Comment:
can you add a uniqueness test

Reply:
Tentatively solved in [37c2305](https://github.com/cal-itp/data-infra/pull/2376/commits/37c2305c46108eae382505d0c71c2696efaec51e), though we'll need to keep an eye out for violations of underlying assumptions here as we get more data, since we don't have a dictionary to work with with uniqueness guarantees laid out.